blocks/rss-fbia-feature-block/features/rss/__snapshots__/xml.test.js.snap

```diff
@@ -11,7 +11,101 @@ Object {
     "@xmlns:sy": "http://purl.org/rss/1.0/modules/syndication/",
     "channel": Object {
       "atom:link": Object {
-        "@href": "http://demo-prod.origin.arcpublishing.com/arc/outboundfeeds/rss/",
+        "@href": "http://demo-prod.origin.arcpublishing.com/undefined",
```

Comment:
is this `undefined` actually ok?

Reply:
Yes, It used to come from a customField. It now comes from fusions requestUri.  I didn't bother to mock that out so it's undefined.